packages/mobile/ios/Podfile.lock

```diff
@@ -708,4 +713,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 0d3a8018c6a563abba8559d74620bf8b8b370fe8
 
-COCOAPODS: 1.7.5
+COCOAPODS: 1.8.4
```

Comment:
Looks like `pod install` was run instead of `bundle exec pod install`.

Reply:
It was yes! what's the difference?

Reply:
There's a fixed version of CocoaPods in `Gemfile.lock` so all devs are using the same one.
When you run `pod install` it runs your system installed CocoaPods, and when you run `bundle exec pod install` it runs the version specified in `Gemfile.lock` (which is recommended).

Reply:
Fixed